vault/expiration.go

```diff
@@ -169,6 +175,22 @@ func newRevocationJob(nsCtx context.Context, leaseID string, ns *namespace.Names
 	}, nil
 }
 
+// errIsUnrecoverable returns true if the logical error is unlikely to resolve
+// automatically or with additional retries
+func errIsUnrecoverable(err error) bool {
+	switch {
+	case errors.Is(err, logical.ErrUnrecoverable),
+		errors.Is(err, logical.ErrUnsupportedOperation),
+		errors.Is(err, logical.ErrUnsupportedPath),
+		errors.Is(err, logical.ErrInvalidRequest),
+		errors.Is(err, logical.ErrPermissionDenied),
+		errors.Is(err, logical.ErrMultiAuthzPending):
```

Comment:
I don't think that `ErrPermissionDenied` or `ErrMultiAuthzPending` should ever happen while sending revoke calls to the mount

Reply:
ab9143ab04